src/Util/ClassSourceManipulator.php

```diff
@@ -197,8 +233,36 @@ public function addSetter(string $propertyName, $type, bool $isNullable, array $
         $this->addMethod($builder->getNode());
     }
 
-    public function addMethodBuilder(Builder\Method $methodBuilder)
+    public function addConstructor(array $params = [], string $methodBody = null)
     {
+        if ($this->getConstructorNode()) {
```

Comment:
```suggestion
        if (null !== $this->getConstructorNode()) {
```

This is a type of stronger coding standard we use in Symfony. I don't always do this in my *own* code, but I do use this here.

Reply:
Okay, good to know